apps/ex_nvr/test/ex_nvr/events_test.exs

```diff
@@ -0,0 +1,126 @@
+defmodule ExNVR.EventsTest do
+  use ExNVR.DataCase
+
+  import ExNVR.DevicesFixtures
+  import ExNVR.RecordingsFixtures
+
+  alias ExNVR.{Events}
+  alias ExNVR.Model.LPREvent
+
+  @moduletag :tmp_dir
+  @lpr_plate "01-D-12345"
+  @lpr_direction "forward"
+  @lpr_time ~U"2023-12-12T10:00:00Z"
+  @lpr_list_type "visitor"
+  @lpr_confidence 0.7
+  @lpr_vehicle_type "Bus"
+  @lpr_vehicle_color "red"
+  @lpr_plate_color "white"
+  @lpr_coordinates_x1 "0"
+  @lpr_coordinates_y1 "100"
+  @lpr_coordinates_x2 "0"
+  @lpr_coordinates_y2 "50"
+  @lpr_plate_image "../fixtures/images/license-plate.jpg"
+                   |> Path.expand(__DIR__)
+                   |> Base.encode64()
+  @lpr_full_image "../fixtures/images/license-plate.jpg"
```

Comment:
Why this images encoded in `Base64` ?? the events context should receive image in binary not in base 64

Reply:
The issue says
```json
{
    "plate_image": "Base64 encoded image",
    "full_image": "Base64 encoded image"
}
```

Isn't this the case ?